pkg/controller/clone-controller.go

```diff
@@ -730,17 +730,15 @@ func ParseCloneRequestAnnotation(pvc *corev1.PersistentVolumeClaim) (exists bool
 	var ann string
 	ann, exists = pvc.Annotations[cc.AnnCloneRequest]
 	if !exists {
-		return
+		return false, "", ""
```

Comment:
Unused named return parameters

Reply:
Same thing, I kept them to avoid confusions about name/namespace ordering